install/operator/pkg/syndesis/action/upgrade.go

```diff
@@ -75,6 +79,24 @@ func (a *upgradeAction) Execute(ctx context.Context, syndesis *v1alpha1.Syndesis
 		if syndesis.Status.Version != targetVersion {
 			a.log.Info("Upgrading syndesis resource ", "name", syndesis.Name, "currentVersion", syndesis.Status.Version, "targetVersion", targetVersion)
 
+			kd := &v14.DeploymentConfig{}
+			err := a.client.Get(ctx, util.NewObjectKey("komodo-server", syndesis.Namespace), kd)
+			if err == nil {
+				a.log.Info("Deleting old komodo deployment", "name", "komodo-server")
```

Comment:
This was `syndesis-dv` in 1.8.x:

https://github.com/syndesisio/syndesis/blob/4aedec35d98f9d496e0074f76d5e3075a82accbc/install/operator/pkg/generator/assets/addons/komodo/addon-komodo-server.yml.tmpl#L27